numpy/matrixlib/defmatrix.py

```diff
@@ -783,7 +783,7 @@ def max(self, axis=None, out=None):
 
     def argmax(self, axis=None, out=None):
         """
-        Indices of the maximum values along an axis.
+        Index of the maximum value along an axis, if multiple returns the first one
```

Comment:
Line length <= 75 characters. The first line is also the summary line and stands alone. I'd do something like
```
        Indexes of the maximum values along an axis.

        Return the indexes of the first occurrences of the maximum values
        along the specified axis.  If axis is None, the index is for
        the flattened matrix.
```